src/test/groovy/com/palantir/gradle/javadist/JavaDistributionPluginTests.groovy

```diff
@@ -201,21 +201,41 @@ class JavaDistributionPluginTests extends GradleTestSpec {
         startScript.contains('DEFAULT_JVM_OPTS=\'"-Djava.security.egd=file:/dev/./urandom" "-Djava.io.tmpdir=var/data/tmp" "-Xmx4M" "-Djavax.net.ssl.trustStore=truststore.jks"\'')
     }
 
-    def 'produce distribution bundle that populates config.sh' () {
+    def 'produce distribution bundle that populates launcher.yml' () {
         given:
         createUntarBuildFile(buildFile)
         buildFile << '''
-            distribution {
-                javaHome 'foo'
+        distribution {
+            javaHome 'foo'
+        }'''.stripIndent()
+        file('src/main/java/test/Test.java') << '''
```

Comment:
Want to make sure the classpath is populated correctly, so need to either add a dependency or this file. I guess I should really add both, in fact.